### DISCUSSION TITLE
How to add other translation languages without registration?

### DISCUSSION THREAD

**Discussion by snmin** [YetiForceCompany/YetiForceCRM#15728] [How to]
I have already submitted the registration information, but I want to try it out now. The default English language is very uncomfortable to use, and I want to switch to the local language. However, it is prompted that the translation language pack cannot be uploaded without registration. In addition, submitting a new language on the page always prompts: "Language tag not compliant with the IETF standard." Failed to add. Is there a proper way to add new languages?

**Comment by snmin** [comment]
I have already submitted the registration information, but I want to try it out now. The default English language is very uncomfortable to use, and I want to switch to the local language. However, it is prompted that the translation language pack cannot be uploaded without registration. In addition, submitting a new language on the page always prompts: "Language tag not compliant with the IETF standard." Failed to add. Is there a proper way to add new languages?

**Comment by snmin** [comment]
In addition, I would like to ask how to register quickly. In my previous installation, I reset the server before waiting for registration activation. Later, I received the information that the registration passed by email. Now the reinstallation prompt still needs to be re registered, and the product key & app ID is different from that before. Can this be registered?

**Comment by rskrzypczak** [comment]
I don't know what your problem is here.
The license requires you to register the system, so the first thing you should do after installing the system is to register it.
[LINK](https://yetiforce.com/en/marketplace/system-registration)
You must follow the same registration steps with each new installation.
A little note - make sure cron works for you.

**Comment by snmin** [comment]
我已经修改好语言，在测试使用的时候发现有些页面显示错误。TaxConfiguration
***@***.***

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: Radosław ***@***.***>
发送时间: 2022年2月18日 13:53
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [YetiForceCompany/YetiForceCRM] How to add other translation languages without registration? (Discussion #15728)


I don't know what your problem is here.
The license requires you to register the system, so the first thing you should do after installing the system is to register it.
LINK<https://yetiforce.com/en/marketplace/system-registration>
You must follow the same registration steps with each new installation.
A little note - make sure cron works for you.

**Reply by snmin** [reply]
我已经修改好语言，在测试使用的时候发现有些页面显示错误。TaxConfiguration

**Reply by snmin** [reply]
![image](https://user-images.githubusercontent.com/44972156/154629354-f79608fc-d1de-43e5-8a63-a39c15b92a67.png)

**Reply by snmin** [reply]
![image](https://user-images.githubusercontent.com/44972156/154629505-7ec58f94-ae5d-4ed4-92a7-52545a705280.png)

**Reply by rskrzypczak** [reply]
I don't know what you're up to, but you can always check the correctness of the displayed content on the demo version (https://github.com/YetiForceCompany/YetiForceCRM#-demos).
Keep in mind that the default language in here is English.

**Comment by snmin** [comment]
Sorry, the question is this. When using Chinese, an error will be reported when opening the "Software configuration >Advanced modules>Tax configuration" interface. Displays fine when using English.

**Comment by snmin** [comment]
Sorry, the question is this. When using Chinese, an error will be reported when opening the "Software configuration >Advanced modules>Tax configuration" interface. Displays fine when using English.

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: Radosław ***@***.***>
发送时间: 2022年2月18日 14:35
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [YetiForceCompany/YetiForceCRM] How to add other translation languages without registration? (Discussion #15728)


I don't know what you're up to, but you can always check the correctness of the displayed content on the demo version (https://github.com/YetiForceCompany/YetiForceCRM#-demos).
Keep in mind that the default language in here is English.
